sources/packages/api/src/esdc-integration/msfaa-integration/msfaa-files/msfaa-file-header.ts

```diff
@@ -1,30 +1,27 @@
-import { StringBuilder } from "../../utilities/string-builder";
 import {
   DATE_FORMAT,
   MSFAARequestFileLine,
   MSFAA_SENT_TITLE,
   SPACE_FILLER,
   TIME_FORMAT,
-  TransactionCodes,
+  RecordTypeCodes,
 } from "../models/msfaa-integration.model";
-import { getDateOnlyFromFormat } from "../../utilities";
+import { StringBuilder, getDateOnlyFromFormat } from "../../../utilities";
 
 /**
  * Header of a MSFAA request file.
  * The documentation about it is available on the document
  * 'CSLP-AppendixF2AsReviewed2016-FileLayouts BC Files V3(HAJ-CB EDITS) In ESDC Folder'.
  */
 export class MSFAAFileHeader implements MSFAARequestFileLine {
-  transactionCode: TransactionCodes;
+  transactionCode: RecordTypeCodes;
   processDate: Date;
-  provinceCode: string;
-  environmentCode: string;
   sequence: number;
 
   public getFixedFormat(): string {
     const header = new StringBuilder();
     header.append(this.transactionCode);
-    header.appendWithEndFiller(this.provinceCode, 4, SPACE_FILLER);
+    header.appendWithEndFiller("BC", 4, SPACE_FILLER);
```

Comment:
Y we are hardcoding the province code? I know we are developing only for the BC province, but why cant we take it from the env variables?

Reply:
So, previously there was a province env variable that was being used to add the province code and also the originator code.
The initial idea was to have both added as env variables, but after considerations and checking how BC and originator are likely to change, the final decision was to not have those env variables due to the lack of a strong reason.

Reply:
As we talked, I just moved it to a constant.